lib/httparrot/request_store.ex

```diff
@@ -0,0 +1,20 @@
+defmodule HTTParrot.RequestStore do
+
```

Comment:
I wonder if you could add a `@moduledoc` to the module, @doc and @spec to the functions?